webrtc.html

```diff
@@ -10618,21 +10618,22 @@ <h3>MediaStreamTrack</h3>
       <p>A <code>MediaStreamTrack</code> object's reference to its
       <code>MediaStream</code> in the non-local media source case (an RTP
       source, as is the case for <code>MediaStreamTrack</code>s received over
-      an <code><a>RTCPeerConnection</a></code> ) is always strong.</p>
+      an <code><a>RTCPeerConnection</a></code>) is always strong.</p>
       <p>When an <code><a>RTCPeerConnection</a></code> receives data on an RTP
-      source for the first time, it MUST <a>update the muted state</a> of the
-      corresponding <code><a>MediaStreamTrack</a></code> with the value
-      <code>false</code>.</p>
+      source for the first time, it MUST queue a task to
+      <a>set the muted state</a> of the corresponding
+      <code><a>MediaStreamTrack</a></code> to <code>false</code>.
+      </p>
       <p>When one of the SSRCs for RTP source media streams received
       by an <code><a>RTCPeerConnection</a></code> is removed (either
-      due to reception of a BYE or via timeout), it MUST <a>update
-      the muted state</a> of the corresponding
-      <code><a>MediaStreamTrack</a></code> with the value
-      <code>true</code>. If and when packets are received again,
-      the <a data-lt="update the muted state">muted state MUST be
-      updated</a> with the value <code>false</code>.</p>
-      <p>The procedure <dfn data-lt="update the muted state" id=
-      "update-track-muted">update a track's muted state</dfn> is specified in
+      due to reception of a BYE or via timeout), it MUST queue a task to
+      <a>set the muted state</a> of the corresponding
+      <code><a>MediaStreamTrack</a></code> to
+      <code>true</code>. If and when packets are received again, it must queue a
+      task to <a data-lt="update the muted state">set the muted state</a> back
+      to <code>false</code>.</p>
+      <p>The procedure <dfn data-lt="set the muted state" id=
+      "update-track-muted">set a track's muted state</dfn> is specified in
```

Comment:
should this "update" also be changed to "set"?

Reply:
Yes thanks for spotting that!